lighthouse-core/lib/proto-preprocessor.js

```diff
@@ -70,6 +70,7 @@ function processForProto(lhr) {
 
   // Drop the i18n icuMessagePaths. Painful in proto, and low priority to expose currently.
   if (reportJson.i18n && reportJson.i18n.icuMessagePaths) {
+    // @ts-expect-error
```

Comment:
why this? should we set it to undefined or do we need to update the type to allow it to be undefined to reflect reality?

Reply:
done.

And now I realize the #10148 wouldn't work for PSI LHRs until we fix this :(